src/frontend/src/components/deviceListItem.ts

```diff
@@ -21,42 +36,45 @@ export type DedupDevice = Device & { dupCount?: number };
 export const deviceListItem = ({
   device,
   index,
+  block,
 }: {
   device: DedupDevice;
   index: string;
+  block?: TemplateResult;
 }) => {
   return html`
     <li class="c-action-list__item" data-device=${device.label}>
+      ${device.status !== undefined
+        ? html`<div class="c-action-list__action">
+            <span
+              class="c-tooltip c-icon c-icon--${device.status.statusType}"
```

Comment:
would it be worth to add a comment somewhere to specify that the type matches a CSS style? (to avoid changing one and not the other in the future)

Reply:
ah yes definitely!

Reply:
I added a comment on the type definition